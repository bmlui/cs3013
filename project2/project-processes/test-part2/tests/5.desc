Ensure shell waits for command to finish before displaying next prompt 
