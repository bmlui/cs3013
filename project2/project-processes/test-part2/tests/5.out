test-part2$ sleep 3
test-part2$ exit
